frontend/lib/norent/about-your-letter.tsx

```diff
@@ -0,0 +1,94 @@
+import { NorentFaqsPreview } from "./faqs";
+import { StaticImage } from "../ui/static-image";
+import { LetterPreview } from "../static-page/letter-preview";
+
+export const NorentAboutYourLetterPage: React.FC<{}> = () => (
+  <Page title="NoRent | About Your Letter" className="content">
+    <section className="hero">
+      <div className="hero-body">
+        <div className="container jf-has-text-centered-tablet">
+          <h2 className="title is-spaced has-text-info">About Your Letter</h2>
+          <br />
+          <p className="subtitle">
+            Not being able to pay rent due to COVID-19 is nothing to be ashamed
+            of. Our letter builder makes it easy to send a letter to your
+            landlord.
+          </p>
+          <br />
+          <BuildMyLetterButton isHiddenMobile />
+          <br />
+        </div>
+      </div>
+      <div className="container jf-has-centered-images jf-space-below-2rem">
+        <JumpArrow to="#more-info" />
+      </div>
+    </section>
+
+    <StickyLetterButtonContainer containerId="more-info">
+      <section className="hero">
+        <div className="hero-body">
+          <div className="container jf-tight-container jf-has-centered-images jf-space-below-2rem">
+            <StaticImage
+              ratio="is-128x128"
+              src={getImageSrc("question")}
+              alt=""
+            />
+            <h2 className="title is-spaced">Why send a letter</h2>
+            <p className="subtitle is-size-4">
+              Here are a few benefits to sending a letter to your landlord:
+            </p>
+            <br />
+            <ul>
+              <li>
+                <p className="title is-size-6">Exercise your rights</p>
+                <p>
+                  Benefit from the eviction protections that local elected
+                  officials have put in place, by notifying your landlord of
+                  your inability to pay rent for reasons related to COVID-19
+                </p>
+              </li>
+              <br />
+              <li>
+                <p className="title is-size-6">Establish your defense</p>
+                <p>
+                  In the event that your landlord tries to evict you, the courts
+                  will see this as a proactive step that helps establish your
+                  defense.
+                </p>
+              </li>
+            </ul>
+            <br />
+            <BuildMyLetterButton isHiddenMobile />
+          </div>
+        </div>
+      </section>
+
+      <LandingPageChecklist />
+
+      <section className="hero">
+        <div className="hero-body">
+          <div className="container has-text-centered">
+            <h3 className="is-spaced has-text-weight-normal">
+              Here’s what the letter will look like:
+            </h3>
+            <br />
+            <LetterPreview
+              title="Preview of your NoRent.org letter"
+              src="/sample-letter.html"
```

Comment:
Oh can you change this to `Routes.locale.sampleLetterContent.html`?  Aside from being more DRY, it will also make it so that it will show the properly localized version of the sample letter once we localize the app!

Reply:
Another thing I'm curious about is if we do want to link to the PDF at some point.  I'm curious if users perceive PDFs as slightly more "real" than the HTML versions just because they have a concept of a printed page about them, but that's a question for Tahnee and it makes no difference to this review.

Reply:
oh cool! Ok all set. And hmmm, yeah I'll bring that up with Tahnee and see what she says.